src/Dfc.CourseDirectory.Web/Views/Apprenticeships/Complete/Index.cshtml

```diff
@@ -1,33 +1,38 @@
-﻿@using Microsoft.AspNetCore.Authorization;
-@using Microsoft.AspNetCore.Identity
-@using Dfc.CourseDirectory.Services.Models
-@model Dfc.CourseDirectory.Web.ViewModels.Apprenticeships.CompleteViewModel
+﻿@model Dfc.CourseDirectory.Web.ViewModels.Apprenticeships.CompleteViewModel
 @{
-    ViewData["Title"] = Model.ApprenticeshipName + " has been added to the Course directory"; 
-    Layout = "_Layout_Your_Courses";
+    ViewData["Title"] = Model.ApprenticeshipName + " has been added to the Course directory";
+    Layout = "_V2LayoutProviderContext";
 }
 
-<div class="govuk-panel govuk-panel--confirmation">
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <div class="govuk-panel govuk-panel--confirmation">
+            <div class="govuk-panel__body">
+                <h1 class="govuk-panel__title">
+                    Apprenticeship published
+                </h1>
+                <div class="govuk-panel__body">
+                    @Model.ApprenticeshipName
+                </div>
+            </div>
+        </div>
```

Comment:
Would be good to use `govuk-panel` here (e.g. https://github.com/SkillsFundingAgency/dfc-coursedirectory/blob/8ceb51fd5c0f3731b73510029834d12d4d974561/src/Dfc.CourseDirectory.Web/Views/BulkUpload/Complete/Index.cshtml#L8-L12)

Reply:
tag helpers all done, cheers